.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - run:
           name: build Hugo
           environment:
-            HUGO_VERSION: 0.58.3
+            HUGO_VERSION: 0.62.2
```

Comment:
Do we need to do anything to ensure the local developer version is also up-to-date?

Reply:
Building the site doesn't _require_ the higher version. Hence, devs don't need to update.

But there might so some small parts like md rendering that are broken with an old version.